cosmos/__init__.py

```diff
@@ -6,7 +6,7 @@
 Contains dags, task groups, and operators.
 """
 
-__version__ = "1.7.1"
+__version__ = "1.8.0a5"
```

Comment:
```suggestion
__version__ = "1.8.0"
```
Reminder to update version before releasing